openbb_platform/core/openbb_core/app/model/abstract/tagged.py

```diff
@@ -1,10 +1,10 @@
 """OpenBB Core App Abstract Model Tagged."""
 
 from pydantic import BaseModel, Field
-from uuid_extensions import uuid7str  # type: ignore
+from uuid_extensions import uuid7str
 
 
 class Tagged(BaseModel):
```

Comment:
I'm not sure this abstraction is actually needed. It is used in 2 places, works as id boilerplate (so there's not much code reused), but on the other hand it hides the id from the models that inherit from it making it less obvious.

Unless there is an understanding where this model will be used apart from the places where it's used now, I would suggest removing it in subsequent PRs